<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AimDialog</class>
 <widget class="QDialog" name="AimDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="TopBar" name="topBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="LeftBar" name="leftBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>50</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>241</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeWidget::item {
height:37px;
}</string>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="TalkView" name="talkView" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>39</y>
     <width>321</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TopBar</class>
   <extends>QWidget</extends>
   <header>topbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeftBar</class>
   <extends>QWidget</extends>
   <header>leftbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TalkView</class>
   <extends>QWidget</extends>
   <header>talkview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
